pages/standards-guidelines/wai-aria.md

```diff
@@ -99,23 +101,26 @@ Additional changes are in the [changelog](https://www.w3.org/TR/wai-aria-1.3/#ch
 
 ### WAI-ARIA 1.2 {#wai-aria-1_2}
 
-The 1.2 version extends WAI-ARIA 1.1 to add a small number of features to the HTML + ARIA accessibility model. For 1.2, user agent implementation guidance is provided as a suite of accessibility API mapping specifications that describe how to expose semantics of WAI-ARIA and other web content languages to accessibility APIs.
+The 1.2 version extends WAI-ARIA 1.1 to add a small number of features to the HTML and ARIA (HTML + ARIA) accessibility model. For 1.2, user agent implementation guidance is provided as a suite of accessibility API mapping specifications that describe how to expose semantics of WAI-ARIA and other web content languages to accessibility APIs.
+
+Published WAI-ARIA specifications are as follows:
 
-Published WAI-ARIA Specifications are as follows:
+-   **[WAI-ARIA 1.2](/TR/wai-aria-1.2/) technical specification** &mdash; provides features to define accessible user interface elements in order to improve the accessibility and interoperability of web content and applications. It is primarily for developers of Web browsers, assistive technologies, and other user agents; developers of Web technologies (technical specifications); and developers of accessibility evaluation tools.
+-   **[Core Accessibility API Mappings 1.2](/TR/core-aam-1.2/)** &mdash; describes how user agents should expose semantics of web content languages to accessibility APIs. The core module defines support that applies across multiple content technologies, including general keyboard navigation support and mapping of general-purpose WAI-ARIA features; other specifications will extend this for specific technologies.
```

Comment:
```suggestion
-   **[Core Accessibility API Mappings 1.2](/TR/core-aam-1.2/)** &mdash; describes how user agents should expose semantics of web content languages to accessibility APIs. The core module defines support that applies across multiple content technologies, including general keyboard navigation support and mapping of general-purpose WAI-ARIA features; other specifications extend this for specific technologies.
```
I think there are already 'specifications that extend this for other technologies', so no need for future tense.